module.json

```diff
@@ -20,6 +20,9 @@
   ],
   "dependencies": {},
   "targetDependencies": {
+    "st-ble-shield": {
+      "x-nucleo-idb0xa1": "ARMmbed/ble-x-nucleo-idb0xa1"
```

Comment:
I believe this is mis-formatted. You should be providing a version number instead of the location of x-nucleo-idb0xa1